src/code/dialogs/checklist.js

```diff
@@ -119,8 +119,11 @@ const OperationChecklistDialog = WDialog.extend({
           oif.value = value;
           oif.size = 3;
           oif.disabled = !canWrite;
+          oif.type = "number";
```

Comment:
one reason I didn't use `number` for this input type is because of the resulting UI in browsers/android
![image](https://user-images.githubusercontent.com/64744459/164544158-62a8886c-ef48-4709-bfde-27396f479a10.png)